cli/internal/cmd/status.go

```diff
@@ -108,21 +121,44 @@ func status(ctx context.Context, kubeClient kubeClient, helmClient helmClient, d
 		return "", fmt.Errorf("getting cluster status: %w", err)
 	}
 
-	return statusOutput(targetVersions, serviceVersions, status, nodeVersion), nil
+	return statusOutput(targetVersions, serviceVersions, status, nodeVersion, string(prettyYAML)), nil
+}
+
+func convertJSONToYAML(rawJSON []byte, err error) ([]byte, error) {
+	var jsonMap map[string]interface{}
+	err = json.Unmarshal(rawJSON, &jsonMap)
```

Comment:
Instead of unmarshalling to a map, and then marshalling that as yaml, you could unmarshal to an attestation config object directly using [UnmarshalAttestationConfig](https://github.com/edgelesssys/constellation/blob/859dfc309fa34d83a13caa5bb86fac73bb911068/internal/config/attestation.go#L32) from the config package.
The different attestation configs should all implement valid yaml marshaling methods.

This approach would also ensure the output from `constellation status` matches what would be written in a users config, since it would use the same marshaling code.

Reply:
done